cobalt/h5vcc/h5vcc_crash_log.cc

```diff
@@ -18,7 +18,6 @@
 #include <string>
 
 #include "base/atomicops.h"
-#include "base/memory/singleton.h"
```

Comment:
Why remove this? You're still using base::Singleton in this file. Isn't it generally best practice to include what you use?

Reply:
agreed.